catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -73,10 +69,6 @@ private static boolean isAnonymousReportEnabled() {
 		return preferences != null && preferences.getBoolean(SettingsActivity.SETTINGS_CRASH_REPORTS, false);
```

Comment:
I guess @84n4n4 means
```java
private static boolean isReportingEnabled() {
    return isCrashReportEnabled &&  preferences != null && preferences.getBoolean(SettingsActivity.SETTINGS_CRASH_REPORTS, false);
}
```

> This is to prevent the reporting during the development mode

Actually, crashes aren't reported to fabric during development mode anyway because
* Crashlytics is [initialized in development mode](https://github.com/Catrobat/Catroid/blob/5b3c137387d7190c99adab52313e46ea1d0f9931/catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java#L61-L63) (and also disabled through [gradle](https://github.com/Catrobat/Catroid/blob/5b3c137387d7190c99adab52313e46ea1d0f9931/catroid/build.gradle#L90))
* The [real api keys](https://github.com/Catrobat/Catroid/blob/develop/catroid/fabric.properties) aren't commited.

However, as @nishantt95 already mentioned, the standalone version has disabled crash reporting by default (don't ask me why).